Server/Server.Core/Server.Core.Static/ServerBuilder.hpp

```diff
@@ -47,7 +47,7 @@ class ServerBuilder
         {
             if (configName != "--serverport")
             {
-                for (size_t i = 0; i < configName.length(); ++i)
+                for (size_t i = 0; i < configName.length(); i++)
```

Comment:
The prefix increment is a better practice than the postfix.

P.S.: But in this case it's no matter. Because those lines will be removed.

Reply:
to be honest in the course of the experiments I took out this method, accidentally corrected this way:)